### PR TITLE
Move all third party license files into a single directory

### PR DESCRIPTION
**Pull request by dadlerj**
@felixfbecker will this break your script?

@tsenart any update on https://github.com/sourcegraph/sourcegraph/issues/9975? Any chance you could give it a shot tonight? We have a customer that is interested in the GPL components we distribute with (including Linux, Prometheus, etc.). Let me know!

### CODE REVIEW

**Comment by felixfbecker** [comment]
Yeah, you'll need to update `dev/licenses-npm.sh` with the new path

**Comment by tsenart** [comment]
> @tsenart any update on #9975? 

I can whip up a quick non exhaustive CSV of only our infrastructure dependencies and put that up for review. As for automating generating the Go module dependencies, I don't have the bandwidth for this anytime soon. Perhaps this is something @bobheadxi could help with soonish? cc @nicksnyder @slimsag

**Comment by tsenart** [comment]
Pushed a commit with it. PTAL

**Comment by slimsag** [comment]
> As for automating generating the Go module dependencies, I don't have the bandwidth for this anytime soon. Perhaps this is something @bobheadxi could help with soonish? cc @nicksnyder @slimsag

Yes, he could likely work on this in the next few weeks. Can you file an issue with more information and assign to him + label as `team/distribution` and `important`? I'll make sure it gets prioritized appropriately

File: third-party-licenses/ThirdPartyLicensesDistributedTools.csv [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Prometheus,Apache License 2.0,https://github.com/prometheus/prometheus
+Grafana,Apache License 2.0,https://github.com/grafana/grafana
+Jaeger,Apache License 2.0,https://github.com/jaegertracing/jaeger
+PostgreSQL,PostgreSQL License,https://postgresql.org/
+SQLite,SQLite Blessing,https://sqlite.org/
+Redis,3-Clause BSD License,https://redis.io/
+universal-ctags,GNU General Public License v2.0,https://github.com/universal-ctags/ctags
+syntect,MIT License,https://github.com/trishume/syntect
+Zoekt,Apache License 2.0,https://github.com/sourcegraph/zoekt
+comby,Apache License 2.0,https://github.com/comby-tools/comby/
```
Comment (slimsag) [inline third-party-licenses/ThirdPartyLicensesDistributedTools.csv:11]:
```suggestion
Comby,Apache License 2.0,https://github.com/comby-tools/comby/
```

File: third-party-licenses/ThirdPartyLicensesDistributedTools.csv [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Project,License,Repository
+Prometheus,Apache License 2.0,https://github.com/prometheus/prometheus
+Grafana,Apache License 2.0,https://github.com/grafana/grafana
+Jaeger,Apache License 2.0,https://github.com/jaegertracing/jaeger
+PostgreSQL,PostgreSQL License,https://postgresql.org/
+SQLite,SQLite Blessing,https://sqlite.org/
+Redis,3-Clause BSD License,https://redis.io/
+universal-ctags,GNU General Public License v2.0,https://github.com/universal-ctags/ctags
+syntect,MIT License,https://github.com/trishume/syntect
```
Comment (slimsag) [inline third-party-licenses/ThirdPartyLicensesDistributedTools.csv:9]:
```suggestion
```

Comment (slimsag) [reply third-party-licenses/ThirdPartyLicensesDistributedTools.csv:9]: We don't distribute syntect, we build it (and other Rust dependencies, plus many other things) into a Rust service which we distribute: https://github.com/sourcegraph/syntect_server

I will push a dependency file for that.

**Comment by tsenart** [comment]
>  Can you file an issue with more information and assign to him + label as team/distribution and important?

There you go: https://github.com/sourcegraph/sourcegraph/issues/9973

File: third-party-licenses/ThirdPartyLicensesDistributedTools.csv (hunk 2 above)
Comment (dadlerj) [reply third-party-licenses/ThirdPartyLicensesDistributedTools.csv]: @slimsag just to check, should I expect this to arrive before merging or should I file an issue for tracking?

If it won't take long, it'd be nice to have this (even manually generated) in the coming weeks.

Comment (slimsag) [reply third-party-licenses/ThirdPartyLicensesDistributedTools.csv]: Very sorry, Dan. I meant to push this but it turned out to be far more tedious than I thought initially, and I failed to follow up here. Please merge this with the line I indicated above removed, and I will follow up. Filed https://github.com/sourcegraph/sourcegraph/issues/11269

**Comment by bobheadxi** [comment]
I believe we can close this now that we have the new license_finder stuff @ https://github.com/sourcegraph/sourcegraph/pull/11321 - https://github.com/sourcegraph/sourcegraph/tree/master/third-party-licenses

**Pull request closed without merging**